modules/rasBidAdapter.js

```diff
@@ -78,6 +79,13 @@ const getSlots = (bidRequests) => {
   return queryString;
 };
 
+const getGdprParams = (bidderRequest) => {
+  const gdprApplies = utils.deepAccess(bidderRequest, 'gdprConsent.gdprApplies');
+  const consentString = utils.deepAccess(bidderRequest, 'gdprConsent.consentString');
```

Comment:
what if there is no gdpr consent in context?

Reply:
consentString will be undefined. So it will go to request URL. Probably it would be good to ignore this param in request, when it's undefined